src/main/java/org/jboss/tools/intellij/openshift/actions/component/LinkServiceAction.java

```diff
@@ -76,7 +79,7 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
        } else {
           UIHelper.executeInUI(() -> Messages.showWarningDialog("No services to link to", "Link service"));
         }
-      } catch (IOException e) {
+      } catch (IOException | KubernetesClientException e) {
```

Comment:
Not sure why we need this exception but the patter is that the Odo interface throws IOExceptions and implementations should translate KubernetesClientException  into IOException

Reply:
I was getting this exception when trying to link services to components and it wasn't catch. so user was facing a no op whane clicking on the menu. I'll improve to catch only IOE.